docs/limitations.md

```diff
@@ -36,3 +36,15 @@ CSI ephemeral volumes may leave orphaned logical volumes
 
 The logical volume created by CSI ephemeral volumes may be left behind by restarting the node.
 This problem is because the kubelet on the restarted node may fail to remove the logical volume through the CSI driver when pods are removed.
+
+Extra arguments to lvcreate are not checked
+-------------------------------------------
+
+TopoLVM does not check the `lvcreate-options` that can optionally be added to a device-class.
+Therefore it cannot take them into consideration when scheduling, or do sanity checks for them.
+It is up to the user make sure that these arguments make sense and work with the VG in question.
+For example, with `--type=raid1`, the VG must have at least 2 PVs to be able to create any LVs.
+
+Note also that the options may affect the "actual" available capacity.
+With `--type=raid1`, each LV will take up twice the normal space.
+You may want to tweak the `spare-gb` setting to avoid some issues with this.
```

Comment:
It's better to describe what happens if users set `lvcreate-options`, in detail. Or it's also OK to refer to [the design document](https://github.com/topolvm/topolvm/blob/main/docs/proposals/lvcreate-options.md#design-details)

Reply:
Done